File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/utils/Heartbeat.java
package de.markhaehnel.rbtv.rocketbeanstv.utils;

import android.os.Bundle;
import com.google.firebase.analytics.FirebaseAnalytics;

public final class Heartbeat {
    public static void doHeartbeat(FirebaseAnalytics firebaseAnalytics) {
        final String HEARTBEAT = "HEARTBEAT";
        Bundle bundle = new Bundle();
        bundle.putString(HEARTBEAT, HEARTBEAT);
        firebaseAnalytics.logEvent(HEARTBEAT, bundle);
    }
}

File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/utils/Heartbeat.kt
package de.markhaehnel.rbtv.rocketbeanstv.utils

import android.os.Bundle
import com.google.firebase.analytics.FirebaseAnalytics

object Heartbeat {
    fun doHeartbeat(firebaseAnalytics: FirebaseAnalytics) {
        val HEARTBEAT = "HEARTBEAT"
        val bundle = Bundle()
        bundle.putString(HEARTBEAT, HEARTBEAT)
        firebaseAnalytics.logEvent(HEARTBEAT, bundle)
    }
}
